bin/auto_test.sh

```diff
@@ -43,13 +43,15 @@ export git_user=""
 export METviewerTag="HEAD"
 export METviewerCompareTag="HEAD"
 export capture=""
+export load=""
 export PYTHON_ENV=""
 export METCALCPY_HOME=""
 export METPLOTPY_HOME=""
+export METDATADB_HOME=""
 
-while getopts "U:t:b:B:l:d:m:a:g:G:u:p:h:P:s:j:c:T:E:A:R?" o; do
+while getopts "U:t:b:B:l:d:m:a:g:G:u:p:h:P:s:j:c:T:E:A:R:D:L:?" o; do
     case "${o}" in
-    	c)
+    	  c)
```

Comment:
```suggestion
        c)
```
Just consistent spacing.